lib/addons/p5.dom.js

```diff
@@ -550,12 +554,19 @@
    * }
    * </code></div>
    */
+
   p5.prototype.createSelect = function(mult) {
-    var elt = document.createElement('select');
-    if (mult){
-      elt.setAttribute('multiple', 'true');
+
+    if( typeof mult === 'object' && mult.elt.nodeName == 'SELECT' ){
+      var self = mult;
+      var elt = this.elt = mult.elt;
+    }else{
+      var elt = document.createElement('select');
+      if( mult && typeof mult === 'boolean' ){
+        elt.setAttribute('multiple', 'true');
+      }
+      var self = addElement(elt, this);
```

Comment:
This is a local variable, so the self.option may not attach correctly. Instead `var self;` should e declared before the if/else, then assigned in each part.